ot3_state_manager/ot3_state_manager/pipette_model.py

```diff
@@ -0,0 +1,15 @@
+"""Enum containing valid pipette models."""
+from enum import Enum
+
+
+class PipetteModel(Enum):
+    """Enum containing valid pipette models for OT3."""
+
+    # TODO: I know these aren't the correct pipette models
+    # Need to update when there is more clarity on what they are
+    # Maybe pull from shared data in monorepo?
```

Comment:
pulling them from shared data is almost certainly the right thing, but yeah this is definitely not something we need for this pr.